geoviews/plotting/bokeh/callbacks.py

```diff
@@ -118,7 +122,8 @@ class GeoRangeXYCallback(RangeXYCallback):
 
     def _process_msg(self, msg):
         msg = super()._process_msg(msg)
-        return project_ranges(self, msg, ('x_range', 'y_range'))
+        ranges = project_ranges(self, msg, ('x_range', 'y_range'))
+        return ranges
```

Comment:
Doesn't seem to add anything?

Reply:
Oh yeah, I can revert that; it was mainly to printing it out during debugging.